extensions/proposals/WEBGL_video_texture/extension.xml

```diff
@@ -20,14 +20,21 @@
   <overview>
     <mirrors href="https://www.khronos.org/registry/gles/extensions/OES/OES_EGL_image_external.txt"
              name="OES_EGL_image_external">
-      <addendum>Defines a new texture target <code>TEXTURE_VIDEO_IMAGE</code>.</addendum>
+      <addendum>Defines a new texture target <code>TEXTURE_VIDEO_IMAGE_WEBGL</code> and a new sampler type <code>GL_SAMPLER_VIDOE_IMAGE_WEBGL</code></addendum>
       <addendum>Provides a mechanism for binding <code>HTMLVideoElement</code> stream to video texture targets.</addendum>
       <addendum>Provides time of frame, texture width and height of <code>HTMLVideoElement</code>'s texture binding.</addendum>
     </mirrors>
 
     <features>
       <feature>Add support for <code>WEBGL_video_texture</code>
       binding of HTMLVideoElement.</feature>
+      <feature>Bind textures to <code>TEXTURE_VIDEO_IMAGE_WEBGL</code> may unbind textures binded to <code>TEXTURE_2D</code>.</feature>
+      <feature>Textures bind to <code>TEXTURE_VIDEO_IMAGE_WEBGL</code> only accept <code>GL_LINEAR</code> and <code>GL_NEAREST</code>
+        for <code>TEXTURE_MIN_FILTER</code> and <code>TEXTURE_MAG_FILTER</code>.</feature>
+      <feature>Textures binds to <code>TEXTURE_VIDEO_IMAGE_WEBGL</code> only accept <code>GL_CLAMP_TO_EDGE</code> for
+        <code>GL_TEXTURE_WRAP_S</code> and <code>GL_TEXTURE_WRAP_T</code>.</feature>
```

Comment:
Same in this new text, the GL_ prefix should be removed.

Reply:
Done.